packages/pluggableWidgets/html-node-web/src/HTMLNode.editorConfig.ts

```diff
@@ -0,0 +1,185 @@
+import { AttributeValueTypeEnum, HTMLNodePreviewProps } from "../typings/HTMLNodeProps";
+import { hideNestedPropertiesIn, hidePropertiesIn, Problem, Properties } from "@mendix/pluggable-widgets-tools";
+
+type TagAttributeValuePropName = keyof HTMLNodePreviewProps["attributes"][number];
+
+const voidElements = [
+    "area",
+    "base",
+    "br",
+    "col",
+    "embed",
+    "hr",
+    "img",
+    "input",
+    "link",
+    "meta",
+    "source",
+    "track",
+    "wbr",
+    // react specific, it uses `value` prop
+    "textarea"
+];
+
+const disabledElements = ["script"];
+
+function isValidHtmlTagName(name: string): boolean {
+    // rather naive approach that works in most cases
+    return /^[a-z_][\w.-]*$/i.test(name);
+}
```

Comment:
```
isValidHtmlTagName("____bmsdfnvd_-2323e-2n3") // => true
```
Is it expected behavior?

Reply:
The leading underscore is too much for html of a freedom indeed (fine on xml though to start with _). For the rest, we don't want to restrict it too much as it might be used with web components.